app/models/rules_of_origin/steps/product_specific_rules.rb

```diff
@@ -20,12 +18,12 @@ def rules
         subdivided_rules? ? subdivision_rules : all_rules
       end
 
-    private
-
-      def commodity
-        @commodity ||= Commodity.find(commodity_code)
```

Comment:
This means we're dropping the 'cache in model instance' - not sure if this is a consideration but if there are multiple attributes request from a GoodsNomenclature (or the same multiple times) this will result in multiple API requests.

At somepoint we should look to cache api requests but for now maybe add an `||=` into `Base#find_declarable` ?